### DISCUSSION TITLE
Use split-window view

### DISCUSSION THREAD

**Discussion by yangqoor** [sumatrapdfreader/sumatrapdf#3281] [Q&A]
[#1575](https://github.com/sumatrapdfreader/sumatrapdf/issues/1575)[#2509 ](https://github.com/sumatrapdfreader/sumatrapdf/issues/2509)I know there have been some similar issues here before, but the multi-window approach mentioned earlier is not convenient.
Just as described on this URL. [Use split-window view](https://helpx.adobe.com/acrobat/using/adjusting-pdf-views.html)
![img](https://raw.githubusercontent.com/yangqoor/md_images/master/img/2023-02-17_195131.png)
Is it possible to implement such a feature? Instead of implementing it with multiple windows.

**Comment by dungsaga** [comment]
You can use windows snap shortcuts (Win+Left, Win+Right) or just drag a window to the edge of screen. This can split the screen into 2 halves.

Afterwards, you can also use Win+Up or Win+Down to reduce the snapped window further to a quarter of the screen.
